src/analysis/normed_space/lp_space.lean

```diff
@@ -628,6 +628,85 @@ end
 
 end normed_space
 
+section non_unital_normed_ring
+
+variables {I : Type*} {B : I → Type*} [Π i, non_unital_normed_ring (B i)]
+
+lemma _root_.mem_ℓp.infty_mul {f g : Π i, B i} (hf : mem_ℓp f ∞) (hg : mem_ℓp g ∞) :
+  mem_ℓp (f * g) ∞ :=
```

Comment:
Can you also prove the following?
```suggestion
lemma _root_.mem_ℓp.infty_pow {f : Π i, B i} (n : \N) (hf : mem_ℓp f ∞) :
  mem_ℓp (f ^ n) ∞ :=
```

Reply:
I'm not seeing an instance of has_pow S ℕ⁺ with S being a semigroup and ℕ⁺ = {1,2,3,..} anywhere. If such a thing doesn't exist I could just prove this in the unitial case.

Reply:
I'm referring to the case when `B i` is a monoid, sorry for not matking that clear.